openid-sharedsignals-framework-1_0.md

```diff
@@ -601,6 +618,13 @@ default_subjects
     to be transmitted. The Receiver MAY remove subjects added this way via the
     `remove_subject_endpoint`.
 
+stream_ttl
```

Comment:
can we make it `stream_ttl_seconds`? will save people working with the api a documentation lookup